code/modules/mob/living/living.dm

```diff
@@ -659,7 +659,7 @@
 		makeTrail(newloc, T, old_direction)
 
 /mob/living/proc/makeTrail(turf/target_turf, turf/start, direction)
-	if(!has_gravity())
+	if(!has_gravity() || movement_type & THROWN)
```

Comment:
```suggestion
	if(!has_gravity() || (movement_type & THROWN))
```
Just to be safe I would do something like this, bit checks not in brackets can get funky